app-misc/neofetch/neofetch-3.3.0.recipe

```diff
@@ -0,0 +1,38 @@
+SUMMARY="A CLI system information tool written in BASH that supports displaying images"
+DESCRIPTION="Neofetch displays information about your system next to an image, your OS logo, or any ASCII file of your choice. The main purpose of Neofetch is to be used in screenshots to show other users what OS/Distro you're running, what Theme/Icons you're using etc"
+HOMEPAGE="https://github.com/dylanaraps/neofetch"
+COPYRIGHT="2016-2017 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dylanaraps/$portName/archive/$portVersion/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="4808e76bd81da3602cb5be7e01dfed8223b1109e2792755dd0d54126014ee696"
+
+PATCHES="neofetch-3.3.0.patchset"
+
+ARCHITECTURES="?x86 ?x86_gcc2 x86_64"
+
+PROVIDES="
+	neofetch
+	cmd:neofetch
+	"
+REQUIRES="
+	haiku
+	cmd:bash
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	cmd:make
+	"
+GLOBAL_WRITABLE_FILES="settings/neofetch/config.conf"
+
+USER_SETTINGS_FILES="settings/neofetch/config.conf template data/neofetch/config.conf"
+
+INSTALL()
+{
+	make PREFIX= BINDIR=$binDir DATADIR=$dataDir \
```

Comment:
Why no BUILD()?

Reply:
Because there's nothing to build.
https://github.com/dylanaraps/neofetch/blob/master/Makefile#L6

Reply:
I see. I would still create a BUILD() section with a lone ":" in it. But let the others explain their ideas.

I find the Makefile solution really overkill here. One can replicate it with simple bash commands, and it would be still portable. And it wouldn't need make as build dependency.